swagger/forward/swagger.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Forward Server API
+  description: |
+    API for the server listening to the remote instances' push of messages and forwarding them to MailGun.
```

Comment:
relays messages coming from multiple message channels to a single Mailgun email channel.

The clients need to authenticate by specifying their assigned API token.